apps/docs/content/guides/realtime/authorization.mdx

```diff
@@ -123,27 +123,51 @@ exists (
 
 Then, to join a topic with RLS enabled, instantiate the Channel with the `private` option set to `true`.
 
-```typescript
-import { createClient } from 'npm:@supabase/supabase-js@2.38.5'
-const url = 'https://<project_ref>.supabase.com'
-const apikey = '<api_key>'
-
-const client = createClient(url, apikey)
```

Comment:
I see that some code samples on the Realtime guides do include initializing the Supabase client, but not sure if that is necessary, so I deleted it, but let me know if this is good.

Also I spot some code samples on the Realtime guide name the Supabase client variable `client`, but everywhere else on the docs it's `supabase`.